### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by BenchR267**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -99,7 +99,7 @@ PalauDefaults.name.value = "Iam a great String value!"
 ### Get
 Getting your value back is as easy as:
 ```swift
-/// name is an Optional<String>?
+/// name is an String?
```
Comment (madhavajay) [inline README.md]: 'an' will become 'a' String

**Comment by madhavajay** [comment]
Thinking about this, its more clear if it says Optional<String> e.g.
Optional of Type String, because String? could be read as String (question mark) ?

Can you adjust your PR to
-/// name is an Optional<String>?
+/// name is an Optional<String>

**Comment by BenchR267** [comment]
You mean `/// name is an Optional<String>` ?

**Comment by madhavajay** [comment]
Perfect!

**Pull request merged**